container/network.go

```diff
@@ -0,0 +1,62 @@
+package container
+
+import (
+	godocker "github.com/fsouza/go-dockerclient"
+)
+
+// CreateNetwork creates a Docker Network with a namespace
+func CreateNetwork(namespace []string) (network *godocker.Network, err error) {
+	network, err = FindNetwork(namespace)
+	if network != nil || err != nil {
+		return
+	}
+	namespaceFlat := Namespace(namespace)
+	client, err := Client()
+	if err != nil {
+		return
+	}
+	network, err = client.CreateNetwork(godocker.CreateNetworkOptions{
+		Name:           namespaceFlat,
+		CheckDuplicate: true, // Cannot have 2 network with the same name
+		Driver:         "overlay",
+		Labels: map[string]string{
+			"com.docker.stack.namespace": namespaceFlat,
+		},
+	})
+	return
+}
+
+// DeleteNetwork deletes a Docker Network associated with a namespace
+func DeleteNetwork(namespace []string) (err error) {
+	network, err := FindNetwork(namespace)
+	if network == nil || err != nil {
+		return
+	}
+	client, err := Client()
+	if err != nil {
+		return
+	}
+	return client.RemoveNetwork(network.ID)
+}
+
+// FindNetworkStrict finds a Docker Network by a namespace. If no network if found, an error is returned.
+func FindNetworkStrict(namespace []string) (network *godocker.Network, err error) {
```

Comment:
What do you think about this function Strict? the logic is to return an error if the container is nil.

Reply:
Same for this one I would use only the `FindNetwork` that returns an error if the network doesn't exists and we have only one function again 

Also this one is even worst because the `FindNetwork` uses the `FindNetworkStrict` and unset the error... that's a weird behavior for a function

Reply:
This difference of behavior (compare to FindContainer and FindService) is because of go-dockerclient or maybe directly of docker api. All our Find functions should have the same behavior.

Reply:
So every functions like that should have the same behavior which is to be strict. If we ask to find for something it means it should be here otherwise we should have a search or something like that